assets/index_old.html

```diff
@@ -2618,6 +2618,20 @@ <h3 class="project-title"><a href="https://github.com/kunjgit/GameZone/tree/main
         </li>
 
 
+        <li class="project-item active" data-filter-item data-category="open source">
+          <a href="./Games/Pop The Baloon/">
+            <figure class="project-img">
+              <div class="project-item-icon-box">
+               
+              <img src="\GameZone\Pop The Baloons.png" alt="game thumbnail" loading="lazy">
+            </figure>
+            <h3 class="project-title"><a href="https://github.com/codewithganeshhh/GameZone/blob/Work/Games/Pop%20The%20Baloon/index.html"
+                target="_blank">137.Pop The Baloon 🔗</a></h3>
+            <p class="project-category">Play and have fun!</p>
+          </a>
+        </li>
+
+
 
```

Comment:
@codewithganeshhh no need to change it over here !
this is older html file just for reference !
make sure you are adding your game in main README file !

Reply:
yes i added